hack/dockerfile/binaries-commits

```diff
@@ -11,5 +11,5 @@ VNDR_COMMIT=c56e082291115e369f77601f9c071dd0b87c7120
 BINDATA_COMMIT=a0ff2567cfb70903282db057e799fd826784d41d
 
 # CLI
-DOCKERCLI_REPO=https://github.com/dperny/cli
-DOCKERCLI_COMMIT=7230906e0e297999eb33da74e0279c5cf41a119e
+DOCKERCLI_REPO=https://github.com/docker/cli
+DOCKERCLI_COMMIT=3dfb8343b139d6342acfd9975d7f1068b5b1c3d3
```

Comment:
Is there a reason this commit is not `docker/cli` master (`eea4a38cd69d47ae6fb8fc02e9fdab0e23e2560e`)? Seems like the last merge was also due for the code freeze.